src/server/services/toerisme.ts

```diff
@@ -0,0 +1,38 @@
+import { requestData } from '../helpers';
+import { getApiConfig } from '../config';
+
+export interface ToerismeDataContent {
+  city: String;
```

Comment:
`String` met een hoofdletter is niet hetzelfde als de typescript `string`. De versie met een hoofdletter representeert de Javascript String class. Dit geldt ook voor `Number` vs typescript `number`